packages/next/src/rsc/handlers/__tests__/previewRouteHandler.ts

```diff
@@ -0,0 +1,15 @@
+import { NextRequest } from 'next/server';
+import { previewRouteHandler } from '../previewRouteHandler';
+
+describe('previewRouteHandler', () => {
+	it.skip('does not accepts POST requests', async () => {
```

Comment:
Having an issue with fetch where instantiating NextRequest is failling, will look into it afterwards.